packages/build-info/src/frameworks/solid.ts

```diff
@@ -4,18 +4,18 @@ export class Solid extends BaseFramework implements Framework {
   readonly id = 'solid-js'
   name = 'SolidJS'
   npmDependencies = ['solid-js']
-  excludedNpmDependencies = ['solid-start']
+  excludedNpmDependencies = ['@solidjs/start']
   category = Category.SSG
 
   dev = {
-    command: 'npm run dev',
+    command: 'vite',
```

Comment:
Let's use `vite dev` here. There's a really nasty bug in build-info that's on my to-do list to write up in Linear that you'll hit otherwise... (this link will only work for you and @pieh: https://netlify.slack.com/archives/C0779CNL7AS/p1726089060435949)